### PR TITLE
Add Boomerang Card from shawyunz

### PR DESCRIPTION
**Pull request by shawyunz**
## Submission description

Add Boomerang Card animation in .Net Maui.
https://github.com/shawyunz/BoomerangCard.Maui/blob/main/demo/demo.gif

### Possible platform improvements

(Missing zindex docs)

### .NET MAUI Beautiful UI Challenge (August 2022)

If you're submitting this during August 2022, you can enter in the [.NET MAUI Beautiful UI Challenge](https://aka.ms/maui/uichallenge)! After submitting your pull request, grab the URL and fill out the quick form at https://www.surveymonkey.com/r/maui-ui-challenge to let us know where to send your stickers!

### CODE REVIEW

**Comment by jfversluis** [comment]
Thank you so much for your contribution Shaw! Is the repo still set to private by any chance? :)

**Comment by shawyunz** [comment]
> Thank you so much for your contribution Shaw! Is the repo still set to private by any chance? :)

Right, updated 😄

File: src/Snppts/Snippets/BoomerangCardMaui.cs [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+        public bool ContainsmacOSSample => false;
+        public Sdk BuiltWith => Sdk.Maui;
+
+        public string Description => "Boomerang animation in .Net Maui. Inspired by the original Swift UI implementation: https://twitter.com/philipcdavis/status/1534192823792128000";
+
+        public IAmAnAuthor AuthorInfo => new ShawYu();
+
+        public IEnumerable<Uri> ImageUris => new List<Uri>
+        {
+            new Uri("https://github.com/shawyunz/BoomerangCard.Maui/blob/main/demo/demo.gif")
```
Comment (jfversluis) [inline src/Snppts/Snippets/BoomerangCardMaui.cs]:
```suggestion
            new Uri("https://github.com/shawyunz/BoomerangCard.Maui/blob/main/demo/demo.gif?raw=true")
```

File: src/Snppts/Authors/ShawYu.cs [hunk 2]
```diff
@@ -0,0 +1,15 @@
+using System;
+using Snppts.Infrastructure;
+
+namespace Snppts.Authors
+{
+    public class ShawYu : IAmAnAuthor
+    {
+        public string FirstName => "Shaw";
+        public string LastName => "Yu";
+        public Uri Website => "https://www.youtube.com/channel/UCmgk0PHlIvJ6TYt54VUEgHQ";
```
Comment (jfversluis) [inline src/Snppts/Authors/ShawYu.cs]:
```suggestion
        public Uri Website => new Uri("https://www.youtube.com/channel/UCmgk0PHlIvJ6TYt54VUEgHQ");
```

**Review by jfversluis — CHANGES_REQUESTED** [review]
Couple of minor things before we can add this beauty!

**Review by jfversluis — APPROVED** [review]
Awesome, again, thank you so much! It looks amazing!

**Pull request merged**